src/options.ts

```diff
@@ -49,14 +50,7 @@ window.onload = () => {
   ).map((dom) => {
     const key = dom.id as keyof Pick<
       StorageSync,
-      | "featuresAssignmentsColoring"
-      | "featuresAutoSaveReports"
-      | "featuresDeadlineHighlighting"
-      | "featuresRemoveConfirmation"
-      | "featuresFilterCourses"
-      | "featuresDragAndDrop"
-      | "featuresReportTemplate"
-      | "featuresRelativeGradesPosition"
+      Exclude<Feature, "featuresDisableForceFileSaving">
     >
```

Comment:
@mkobayashime Why is `featuresDisableForceFileSaving` excluded here?

Reply:
@yudukikun5120 I'm not sure, but perhaps I had forgotten adding it when I implemented the feature. You can simply use `Feature` now.